apps/leaflet_ts/leaflet_ts/static/js/index.js

```diff
@@ -59,12 +59,12 @@ var WMS_URL="//"+location.hostname+"/thredds/wms/ts/"+getParameterByName("id");
 //NSBAS-PARAMS.h5";
 var LAYERS_CONFIG = [
     {
-        "name":"RAW Displacement Time Series (cm)",
+        "name":"RAW Displacement Time Series wrt Satellite LOS (mm)",
         "id":"rawts",
         "colorscalerange":"-100,100"
     },
     {
-        "name":"Filtered Displacement Time Series (cm)",
+        "name":"Filtered Displacement Time Series wrt Satellite LOS (mm)",
         "id":"recons",
```

Comment:
TODO: change it to 
`Filtered Displacement Time Series in the Satellite LOS (mm)`